spec/system/post/show_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+RSpec.describe 'Index posts page', type: :system do
+  before :each do
+    user1 = User.create(
+      id: 10,
+      name: 'Rodrigo',
+      photo: 'https://randomuser.me/api/portraits/men/31.jpg',
+      bio: 'Awesome User',
+      post_counter: 0
+    )
+    post1 = Post.create(id: 1, title: 'Post Number 1',
+                        text: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Doloremque autem ipsa offics.',
+                        likes_counter: 0,
+                        comments_counter: 0,
+                        author: user1)
+    Post.create(id: 2, title: 'Post Number 2', text: 'Lorem Ipsu', likes_counter: 0, comments_counter: 0, author: user1)
+    Post.create(id: 3, title: 'Post Number 3', text: 'Lorem Ipsu', likes_counter: 0, comments_counter: 0, author: user1)
+    Post.create(id: 4, title: 'Post Number 4', text: 'Lorem Ipsu', likes_counter: 0, comments_counter: 0, author: user1)
+    Comment.create(author: user1, post: post1, text: 'Awesome post!')
+    Like.create(author: user1, post: post1)
+  end
+  describe 'index page' do
```

Comment:
- [ ] **[OPTIONAL]** You have a typo error here. Kindly fix it.

Reply:
Excellent, I will fix it